### DISCUSSION TITLE
Can deterministic simulation settings simplify the process? What should I do if it's some other engine in the world?

### DISCUSSION THREAD

**Discussion by PinkFatMan** [jrouwe/JoltPhysics#626] [Q&A]
According to the document https://jrouwe.github.io/JoltPhysics/ In the Deterministic Simulation section, I need to set some compiler settings for the current application. However, I am now integrating Jolt Physics into Unity. Firstly, I couldn't find the relevant/fp: precision setting method for the Unity engine. And if I want to build it, it would be very cumbersome to first export it as a C++project and then set it up. I also studied it but couldn't find a solution that could simplify the process. Do you have any good suggestions?

**Comment by jrouwe** [comment]
The documentation states that you should use ```/fp:precise``` is that not what you're looking for?

I have no experience with Unity myself, so I'm afraid I can't help you with setting it up.
